packages/ui5-info/test/ui5-version-info.test.ts

```diff
@@ -360,6 +361,32 @@ describe('getUI5Versions: npm listed versions', () => {
 });
 
 describe('getLatestUI5Version', () => {
+    /**
+     * Mocks ui5VersionsCache.officialVersions.
+     *
+     * @param {string[]} [officialVersions] - Versions to populate the cache. Defaults to an empty array if not provided.
+     */
+    const mockUi5VersionsCache = (officialVersions?: string[]) => {
+        ui5VersionsCache.officialVersions = officialVersions || [];
+    };
+
+    beforeAll(() => {
+        // Mock the ui5VersionsCache
+        const originalConstants = jest.requireActual('../src/constants');
+        return {
+            ...originalConstants,
+            ui5VersionsCache: {
+                officialVersions: [],
+                snapshotsVersions: [],
+                support: []
+            }
+        };
+    });
+
+    beforeEach(() => {
```

Comment:
Might be worth resetting the cache before each test (rather than once before all tests) so the tests are isolated?